src/utils/components/CloudinitModal/CloudinitForm.tsx

```diff
@@ -64,7 +64,7 @@ const CloudinitForm: React.FC<CloudinitFormProps> = ({
             fieldId={'cloudinit-user'}
             className="kv-cloudint-advanced-tab--validation-text"
             isRequired
-            helperText={t("Default cloudinit username is 'cloud-user'")}
+            helperText={t("Default cloud-init username is 'cloud-user'")}
```

Comment:
should this be `Cloud-init` as all changes in this PR?

Reply:
I think we should use `cloud-init` if the word does not start the sentence as in the link that the bug refers to (https://cloudinit.readthedocs.io/en/latest/index.html)